tests/lib/managers/events-test.js

```diff
@@ -420,4 +428,33 @@ describe('Events', function() {
 				});
 		});
 	});
+
+	describe('getEnterpriseEventStream()', function() {
+
+		var options = {
+			streamPosition: TEST_STREAM_POSITION
+		};
+
+		it('should return event stream from starting stream position when passed stream position', function(done) {
```

Comment:
This test name is incorrect; it should call the callback with the stream, not return it